docs/docs/dev/security/certificates.md

```diff
@@ -122,6 +122,46 @@ You can also provide the CA via environment variable `KUMA_CONTROL_PLANE_CA_CERT
 
 See [Data plane proxy authentication](./dp-auth.md) and [Zone Ingress authentication](./zone-ingress-auth.md).
 
+## Prometheus to control plane communication
+
+It is possible to enabled TLS on the Monitoring Assignment Discovery Service.
+By default it will use the certificate used in `tls-general` but TLS will be disabled.
+You can enable it by using either the environment variable `KUMA_MONITORING_ASSIGNMENT_SERVER_TLS_ENABLED=true` or the yaml config `monitoringAssignmentServer.tlsEnabled=true`.
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes (kumactl)"
+Create a secret in the namespace where the control plane is installed:
+```sh
+kubectl create secret generic general-tls-certs -n <namespace> \
+  --from-file=tls.crt=/tmp/tls.crt \
+  --from-file=tls.key=/tmp/tls.key \
+  --from-file=ca.crt=/tmp/ca.crt
+```
+
+Point to this secret when installing Kuma:
```

Comment:
```suggestion
Point to this secret when installing {{site.mesh_product_name}}:
```
If this is for 2.0. We created this variable (not merged yet) to swap Kuma/Kong Mesh on shared docs.